### PR TITLE
Re-order Build Tasks to publish to Ballerina Central

### PR DESCRIPTION
**Pull request by ayeshLK**

### CODE REVIEW

**Review by ThisaruGuruge — APPROVED** [review]

File: websub-ballerina/build.gradle [hunk 1]
```diff
@@ -169,6 +173,7 @@ task initializeVariables {
 
         if (graph.hasTask(":${packageName}-ballerina:publish")) {
             ballerinaPublish.enabled = true
+            ballerinaTest.enabled = false
```
Comment (aashikam) [inline websub-ballerina/build.gradle:176]: This is covered in line 160. 
```suggestion
```

File: websub-ballerina/build.gradle [hunk 2]
```diff
@@ -365,22 +368,29 @@ publishing {
     }
 }
 
-ballerinaBuild.dependsOn ":${packageName}-native:build"
-ballerinaBuild.dependsOn updateTomlFile
-ballerinaBuild.dependsOn initializeVariables
-ballerinaBuild.dependsOn test
+unpackJballerinaTools.dependsOn copyToLib
```
Comment (ThisaruGuruge) [inline websub-ballerina/build.gradle:371]: Is there a task named `copyToLib`? Shall we revisit these names?

**Comment by aashikam** [comment]
@ayeshLK Can you add this also https://github.com/ballerina-platform/module-ballerina-graphql/pull/90/files

File: websub-ballerina/build.gradle (hunk 2 above)
Comment (ayeshLK) [reply websub-ballerina/build.gradle:370]: There is a task for `copyToLib` in line 198.

**Review by ThisaruGuruge — APPROVED** [review]

**Pull request merged**